src/controllers/commentController.js

```diff
@@ -0,0 +1,54 @@
+import * as ApplicationError from '../utils/errors/applicationsErrors';
+import * as commentService from '../services/commentService';
+import {
+  createdResponse,
+  notFoundResponse,
+  successResponse
+} from '../utils/responseHandler';
+
+export const addComment = async (req, res) => {
+  try {
+    const { tripId } = req.params;
+    const userId = req.user.id;
+    const comment = await commentService.createComment(
+      userId,
+      tripId,
+      req.body.comment
+    );
+    return comment.error
+      ? ApplicationError.validationError(comment.error, res)
+      : createdResponse(res, 'Comment created successfully', comment);
```

Comment:
Remove the comment you created in the response since you have endpoint/path to retrieve it

Reply:
I need to know which comment created for testing purpose

Reply:
On the front side also there could be a case you may need to know which comment was created. i.e, add animation to indicate a newly added comment.